tg/MainScreenListOfGames/ListOfGamesViewController.swift

```diff
@@ -126,14 +126,14 @@ extension ListOfGamesViewController: UITableViewDelegate, UITableViewDataSource{
         
         if searchIsNotEmpty {
             
-            let id = resultsOfSearch[indexPath.row].id ?? 0
+            let id = resultsOfSearch[indexPath.row].id
```

Comment:
It would better remove checking `searchIsNotEmpty` and use only `resultsOfSearch` array each time. 
Once we receive data from BE set ` resultsOfSearch = gameList`. in filtration method if search string is null or empty set  ` resultsOfSearch = gameList`